### ISSUE TITLE
DayPickerSingleDateController aria-label to div with role="region"

### ISSUE THREAD

**Issue by danielbotn** [airbnb/react-dates#1905]
**react-dates version**
e.g. react-dates@20.2.5

**Describe the bug**
We are using the <DayPickerSingleDateController />. Our accessibility software detects the following violation:

The page contains an HTML5 <section> or a WAI-ARIA landmark of the type 'region'. 
These types of landmarks are very general and should be named to help users identify the purpose of the landmark.

How to fix it:
Use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to name the landmark in a way that 
clearly explains the purpose of the landmark to all users.

<div class="DayPicker_focusRegion DayPicker_focusRegion_1" role="region" tabindex="-1" ...

Is there a way to add aria-label to the div above?

**Comment by indiesquidge** [comment]
@danielbotn, it looks like that element applies an `aria-label` based on the `phrases.calendarLabel` prop supplied to it.

https://github.com/airbnb/react-dates/blob/b9ddf40984d38eb5ba19f6cace3a1c104b95a2ac/src/components/DayPicker.jsx#L1181

The `phrases` can be provided to the controller component you're using, and appears to have a default value for `calendarLabel`.

https://github.com/airbnb/react-dates/blob/b9ddf40984d38eb5ba19f6cace3a1c104b95a2ac/src/defaultPhrases.js#L1

Are you providing a `phrases` prop that may override or exclude this in some way?

**Comment by danielbotn** [comment]
@indiesquidge  No I am not using the phrases prop. These are the props I am using for the DayPickerSingleDateController:

      key
      date
      onDateChange
      onNextMonthClick
      onPrevMonthClick
      focused
      onFocusChange
      numberOfMonths
      isDayBlocked
      isOutsideRange
      hideKeyboardShortcutsPanel
      initialVisibleMonth



Storybook does not flag any violations but siteimprove is complaining about this issue, see description above
